internal/grpctest/grpctest.go

```diff
@@ -53,13 +55,18 @@ func RunSubTests(t *testing.T, x interface{}) {
 	setup := getTestFunc(t, xv, "Setup")
 	teardown := getTestFunc(t, xv, "Teardown")
 
+	logger.setLevel(infoLog)
+	logger.setVerbosity(2)
+	grpclog.SetLoggerV2(&logger)
```

Comment:
Could you move this out of `grpctest` and into the packages that use it instead (and the `logger.t=t` line below, by using a `Setup()`)?

I would like to keep this function (if not the whole package) as something usable outside of grpc, since it is otherwise entirely generic.

It's probably OK to keep the `tLogger` type defined in this package - but it will need to be exported since it will need to be used by all the tests individually.

Reply:
I removed the setup from internal/grpctest/grpctest.go. This file is now unchanged from master.